antlr/Expr.g4

```diff
@@ -0,0 +1,54 @@
+grammar Expr;
+
+start
+  : expr EOF
+  ;
+
+expr:   orExpr ;
+
+orExpr
+    :   orExpr '||' andExpr       # Or
+    |   andExpr                   # ToAnd
+    ;
+
+andExpr
+    :   andExpr '&&' notExpr     # And
+    |   notExpr                  # ToNot
+    ;
+
+notExpr
+    :   '!' notExpr             # Not
+    |   relExpr                 # ToRel
+    ;
+
+relExpr
+    :   relExpr op=('<' | '<=' | '==' | '>' | '>=') addSubExpr  # Relational
```

Comment:
looking good! We also probably want the `!=` as well, right?

Reply:
Yup forgot to put that in